include/ui.js

```diff
@@ -205,47 +205,47 @@ var UI;
 
         addMouseHandlers: function() {
             // Setup interface handlers that can't be inline
-            $D("noVNC_view_drag_button").onclick = UI.toggleViewDrag;
-            $D("noVNC_mouse_button0").onclick = function () { UI.setMouseButton(1); };
-            $D("noVNC_mouse_button1").onclick = function () { UI.setMouseButton(2); };
-            $D("noVNC_mouse_button2").onclick = function () { UI.setMouseButton(4); };
-            $D("noVNC_mouse_button4").onclick = function () { UI.setMouseButton(0); };
-            $D("noVNC_keyboard_button").onclick = UI.showKeyboard;
-
-            $D("noVNC_keyboardinput").oninput = UI.keyInput;
-            $D("noVNC_keyboardinput").onblur = UI.hideKeyboard;
-            $D("noVNC_keyboardinput").onsubmit = function () { return false; };
-
-            $D("noVNC_toggleExtraKeys_button").onclick = UI.toggleExtraKeys;
-            $D("noVNC_toggleCtrl_button").onclick = UI.toggleCtrl;
-            $D("noVNC_toggleAlt_button").onclick = UI.toggleAlt;
-            $D("noVNC_sendTab_button").onclick = UI.sendTab;
-            $D("noVNC_sendEsc_button").onclick = UI.sendEsc;
-
-            $D("noVNC_sendCtrlAltDel_button").onclick = UI.sendCtrlAltDel;
-            $D("noVNC_xvpShutdown_button").onclick = function() { UI.rfb.xvpShutdown(); },
-            $D("noVNC_xvpReboot_button").onclick = function() { UI.rfb.xvpReboot(); },
-            $D("noVNC_xvpReset_button").onclick = function() { UI.rfb.xvpReset(); },
-            $D("noVNC_status").onclick = UI.popupStatus;
-            $D("noVNC_popup_status").onclick = UI.closePopup;
-            $D("noVNC_toggleXvp_button").onclick = UI.toggleXvpPanel;
-            $D("noVNC_clipboard_button").onclick = UI.toggleClipboardPanel;
-            $D("noVNC_fullscreen_button").onclick = UI.toggleFullscreen;
-            $D("noVNC_settings_button").onclick = UI.toggleSettingsPanel;
-            $D("noVNC_connectPanel_button").onclick = UI.toggleConnectPanel;
-            $D("noVNC_disconnect_button").onclick = UI.disconnect;
-            $D("noVNC_description_button").onclick = UI.toggleConnectPanel;
-
-            $D("noVNC_clipboard_text").onfocus = UI.displayBlur;
-            $D("noVNC_clipboard_text").onblur = UI.displayFocus;
-            $D("noVNC_clipboard_text").onchange = UI.clipboardSend;
-            $D("noVNC_clipboard_clear_button").onclick = UI.clipboardClear;
-
-            $D("noVNC_settings_apply").onclick = UI.settingsApply;
-
-            $D("noVNC_connect_button").onclick = UI.connect;
-
-            $D("noVNC_setting_resize").onchange = UI.enableDisableViewClip;
+            Util.addEvent($D("noVNC_view_drag_button"), 'click', UI.toggleViewDrag);
```

Comment:
just a heads up: this is all just going to need to be in the form of `document.getElementById("xyz").addEventListener('event', some_func)` now.